packages/data-context/src/sources/MigrationDataSource.ts

```diff
@@ -0,0 +1,75 @@
+import path from 'path'
+import type { DataContext } from '..'
+
+type ConfigOptions = {
+  global: Record<string, unknown>
+  e2e: Record<string, unknown>
+  component: Record<string, unknown>
+}
+
+export class MigrationDataSource {
+  constructor (private ctx: DataContext) { }
+
+  async getConfig () {
+    const config = await this.parseCypressConfig()
+
+    return JSON.stringify(config, null, 2)
+  }
+
+  private parseCypressConfig (): Promise<Cypress.ConfigOptions> {
+    const cfgPath = path.join(this.ctx.lifecycleManager?.projectRoot, 'cypress.json')
+
+    return this.ctx.file.readJsonFile(cfgPath)
+  }
+
+  async createConfigString () {
+    const cfg = await this.parseCypressConfig()
+    const pluginsPath = this.getPluginRelativePath(cfg)
+    const blah = Object.entries(cfg).reduce((acc, [key, val]) => {
```

Comment:
Whoops I need to change this in the morning, this was temporary while I couldn't think of a name haha

Reply:
Done!